### PR TITLE
Creates new guide on using swupd-add-pkg method for customization of …

### PR DESCRIPTION
**Pull request by Michael-Vincerra**
…CL. Applies line edits. Adds reference for new guide in

parent toctree maintenance.rst. Adds note to paragraph one.  Applies further line edits.

Signed-off-by: Michael Vincerra <michaelx.vincerra@intel.com>

### CODE REVIEW

**Comment by Michael-Vincerra** [comment]
Note: PR 105 replaces PR 54.

**Comment by tmarcu** [comment]
Please close and then reference the PR so it's not just hanging out as an open PR on the list https://github.com/clearlinux/clear-linux-documentation/pull/54

**Comment by MCamp859** [comment]
@tmarcu Please review the latest commit to see if it addresses your review feedback.

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst [hunk 1]
```diff
@@ -1,150 +1,156 @@
...
-#. Create the workspace:
+      swupd bundle-add mixer
 
-   .. code-block:: console
+#. Create the workspace.
 
-      # swupd bundle-add mixer
+   .. code-block:: bash
+
+      swupd bundle-add mixer
```
Comment (tpleavitt) [inline source/clear-linux/guides/maintenance/swupdaddpkg.rst]: This is the same command that is used in step 2, is this correct for creating the workspace?

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst [hunk 2]
```diff
@@ -1,150 +1,156 @@
...
-   You cannot simply use RPMs from other Linux distros on |CL|.  You must
-   build RPMs specifically for |CL| in order for them to work properly.
-   Follow the `Developer tooling framework for Clear Linux`_ on 
-   on how to build RPMs.  
+   You cannot simply use RPMs from other Linux distros on |CL|. You must build
+   RPMs specifically for |CL| in order for them to work properly. Follow the
+   `Developer tooling framework for Clear Linux`_ instructions on how to build
+   RPMs.
 
 Copy your RPM to the workspace.
```
Comment (tpleavitt) [inline source/clear-linux/guides/maintenance/swupdaddpkg.rst]: Is this a file?  If so, use the :file: reference and call it a file, i.e. :file:`RPM` file

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst [hunk 3]
```diff
@@ -1,150 +1,156 @@
...
+#. Create a bundle with your custom RPM package.
+#. Migrate your |CL| system to your custom mix.
+#. Add your custom bundle to your system.
+#. Optional: Revert your system back to 100% upstream.
 
 Set up the workspace
 ********************
 
-#. Log in and get root privileges: 
+#. Log in and get root privileges.
```
Comment (tmarcu) [inline source/clear-linux/guides/maintenance/swupdaddpkg.rst]: No logging into root :-1: :-1:  Use sudo where it's needed:
`sudo swupd-add-pkg foo bar`

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst [hunk 4]
```diff
@@ -1,150 +1,156 @@
...
-#. Install the mixer bundle to enable mixer:
-   
-   .. code-block:: console 
+#. Install the mixer bundle to enable mixer.
 
-      # swupd bundle-add mixer
+   .. code-block:: bash
 
-#. Create the workspace:
+      swupd bundle-add mixer
```
Comment (tmarcu) [inline source/clear-linux/guides/maintenance/swupdaddpkg.rst]: `sudo swupd bundle-add mixer`

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst [hunk 5]
```diff
@@ -1,150 +1,156 @@
...
 ********************
 
-#. Log in and get root privileges: 
+#. Log in and get root privileges.
 
-   .. code-block:: console
+   .. code-block:: bash
 
-      $ sudo -s
+      sudo -s
```
Comment (tmarcu) [inline source/clear-linux/guides/maintenance/swupdaddpkg.rst]: Remove.

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst (hunk 1 above)
Comment (tmarcu) [reply source/clear-linux/guides/maintenance/swupdaddpkg.rst]: Creating the workspace should be:
`sudo mkdir -p /usr/share/mix/rpms` 
Bundle adding mixer does not create this folder.

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst [hunk 6]
```diff
@@ -1,150 +1,156 @@
...
 ********************************************
 
 Use the :command:`swupd-add-pkg` command to create a bundle with the RPM
 package.
 
-.. code-block:: console
+.. code-block:: bash
 
-   # swupd-add-pkg [RPM] [bundle-name]
+   swupd-add-pkg [RPM] [bundle-name]
```
Comment (tmarcu) [inline source/clear-linux/guides/maintenance/swupdaddpkg.rst]: `sudo swupd-add-pkg [RPM] [bundle-name]`

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst [hunk 7]
```diff
@@ -1,150 +1,156 @@
...
+to your custom mix to make the bundle accessible.
 
-Before your custom bundle can be used, you must “migrate” your |CL| system
-to your custom mix to make them accessible.
+.. code-block:: bash
 
-.. code-block:: console
-   
-   # swupd update --migrate
+   swupd update --migrate
```
Comment (tmarcu) [inline source/clear-linux/guides/maintenance/swupdaddpkg.rst]: `sudo swupd --migrate`

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst [hunk 8]
```diff
@@ -1,150 +1,156 @@
...
 Add your custom bundle to your system
 *************************************
 
 #. Get a listing of your newly-created bundle.
 
-   .. code-block:: console
+   .. code-block:: bash
 
-      # swupd bundle-list -a
+      swupd bundle-list -a
```
Comment (tmarcu) [inline source/clear-linux/guides/maintenance/swupdaddpkg.rst]: `sudo  swupd bundle-list -a
`

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst [hunk 9]
```diff
@@ -1,150 +1,156 @@
...
-   The listing will include all upstream bundles as well.
+   The listing includes all upstream bundles.
 
 #. Add your bundle.
 
-   .. code-block:: console
+   .. code-block:: bash
 
-      # swupd bundle-add [bundle-name]
+      swupd bundle-add [bundle-name]
```
Comment (tmarcu) [inline source/clear-linux/guides/maintenance/swupdaddpkg.rst]: `sudo swupd bundle-add [bundle-name]`

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst [hunk 10]
```diff
@@ -1,150 +1,156 @@
...
+   this command:
 
-   You can also update your system to the latest upstream version by using
-   this command:   
-   
-   .. code-block:: console
+   .. code-block:: bash
 
-      # swupd update
+      swupd update
```
Comment (tmarcu) [inline source/clear-linux/guides/maintenance/swupdaddpkg.rst]: `sudo swupd update`

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst [hunk 11]
```diff
@@ -1,150 +1,156 @@
...
-Should you decide to switch your |CL| system back to the official upstream version, you can do so with this command:
+If you want to revert your |CL| system to the official upstream version,
+use this command:
+
+.. code-block:: bash
 
-.. code-block:: console
-   
-   # swupd verify --fix --force --picky -m [upstream-version-number] -C /usr/share/clear/update-ca/Swupd_Root.pem
+   swupd verify --fix --force --picky -m [upstream-version-number] -C /usr/share/clear/update-ca/Swupd_Root.pem
```
Comment (tmarcu) [inline source/clear-linux/guides/maintenance/swupdaddpkg.rst]: `sudo swupd verify --fix --force --picky -m [upstream-version-number] -C /usr/share/clear/update-ca/Swupd_Root.pem`

File: source/clear-linux/guides/maintenance/swupdaddpkg.rst (hunk 2 above)
Comment (Michael-Vincerra) [reply source/clear-linux/guides/maintenance/swupdaddpkg.rst]: This is a URL for the Github repo: https://github.com/clearlinux/common
This is not a file.

**Pull request merged**